todo_app/app.py

```diff
@@ -1,11 +1,24 @@
-from flask import Flask
 
+from flask import Flask, render_template, request, redirect
+from todo_app.data.session_items import get_items
+from todo_app.data import session_items
```

Comment:
Slightly confusing to be importing session_items two ways.
I prefer the second one, so delete the line `from todo_app.data.session_items import get_items`. It's unused anyway (you access the function via `session_items.get_items` below)